maps/away/sol_shuttle/sol_shuttle_radio.dm

```diff
@@ -0,0 +1,31 @@
+/obj/machinery/telecomms/hub/map_preset/sol_shuttle
+	preset_name = "SEV Quantum"
+
+/obj/machinery/telecomms/receiver/map_preset/sol_shuttle
+	preset_name = "SEV Quantum"
+
+/obj/machinery/telecomms/bus/map_preset/sol_shuttle
+	preset_name = "SEV Quantum"
+
+/obj/machinery/telecomms/processor/map_preset/sol_shuttle
+	preset_name = "SEV Quantum"
+
+/obj/machinery/telecomms/server/map_preset/sol_shuttle
+	preset_name = "SEV Quantum"
```

Comment:
Might want to give it some generic channel name instead? Let alone, ship might not be even called that when it spawns.

Reply:
On it! The SEV Quantum was the placeholder name.